TFT/src/User/Hal/buzzer.c

```diff
@@ -75,19 +75,19 @@ void Buzzer_play(SOUND sound)
 {
   // if (infoSettings.allSound != 1)
     // return;
+
+  if (sound != sound_error && infoSettings.alertSound != 1)
+    return;
+
   switch (sound)
```

Comment:
how will this handle `infoSettings.toastSound`, `infoSettings.touchSound` & `infoSettings.heaterSound` settings?

Reply:
Oops, thanks for  the warning.
Code reverted.